src/dbnode/storage/database.go

```diff
@@ -1172,6 +1178,9 @@ func (d *db) AggregateTiles(
 			zap.String("targetNs", targetNsID.String()),
 			zap.Error(err),
 		)
+		opts.MetricsScope.Counter("aggregation.errors").Inc(1)
+	} else {
+		opts.MetricsScope.Counter("aggregation.success").Inc(1)
```

Comment:
Instead of having two different metrics here, should it be one metric with a success tag?

Reply:
I tried to follow the convention which I saw elsewhere: https://github.com/m3db/m3/blob/gg/large-tiles-metrics/src/x/instrument/methods.go#L298
So, which one we should follow? :)